great_expectations/core/usage_statistics/package_dependencies.py

```diff
@@ -142,6 +142,10 @@ class GEDependencies:
         "PyHive",
         "thrift",
         "thrift-sasl",
+        # requirements-dev-test-pipeline.txt:
```

Comment:
These are dependencies we only need when we run in Azure so I am excluding them from tracking. If we want to track these let me know.

Reply:
I think you need to also add these to `ALL_GE_DEV_DEPENDENCIES`

Reply:
Done. I also see that `pytest-cov` is actually in `requirements-dev.contrib.txt` so have removed it from here. Thanks!